docs/shell.rst

```diff
@@ -0,0 +1,141 @@
+=====================================================
+InSpec Shell Usage
+=====================================================
+
+The InSpec interactive shell is a pry based REPL that can be used to quickly run InSpec controls and tests without having to write it to a file.  Its functionality is similar to ``chef shell`` - it provides a way to exercise the InSpec DSL, its resources, tests and plugins without having to create a profile or write a test file.
+
+Launching the shell
+-----------------------------------------------------
+
+If you are using InSpec from a platform-specific package (rpm, msi, etc.) or from a chef shell in ChefDK, you can directly launch chef shell against your local machine using:
+
+.. code-block:: bash
+
+    $ inspec shell
+    $ inspec help shell # This will describe inspec shell usage
+
+If you wish to connect to a remote machine (called a target within InSpec), you can use the ``-t`` flag.  We support connecting using ssh, WinRm and docker.  For an ssh connection, use ``-i`` for specifying ssh key files, and the ``--sudo*`` commands for requesting a privelege escalation after logging in.  For a WinRM connection, use ``--path`` to change the login path, ``--ssl`` to use SSL for transport layer encryption.
+
+.. code-block:: bash
+
+    $ inspec shell -t ssh://root@192.168.64.2:11022  # Login to remote machine using ssh as root.
+    $ inspec shell -t ssh://user@hostname:1234 -i /path/to/user_key  # Login to hostname on port 1234 as user using given ssh key.
+    $ inspec shell -t winrm://UserName:Password@windowsmachine:1234  # Login to windowsmachine over WinRM as UserName.
+    $ inspec shell -t docker://container_id # Login to a docker container.
+
+If you are installing from source using bundler, you may run InSpec within your bundle.  All the above platform commands remain valid.
```

Comment:
I am not sure if this is more confusing for users without extensive ruby knowledge, since they may not know bundler. From end-user perspective you should get InSpec via a package.